### ISSUE TITLE
[BUG] Imgur account authorization fails

### ISSUE THREAD

**Issue by jslpc** [ailinykh/ImgurBar#2]
When building and running ImgurBar, everything launches perfectly. However, trying to login via the preference pane results in the following error from Imgur.com:

![The information you sent wasn't quite correct. Please return to the place that sent you to this page and try again.](https://i.imgur.com/80hBqfU.png)

> The information you sent wasn't quite correct. Please return to the place that sent you to this page and try again.

I managed to fix this by editing line 48 in AppDelegate.swift to not replace the URL with "FAKE_CLIENT_ID", but I don't really know Swift so I'm not sure how to fix this from happening. 

I also ran into a callback-url error while logging in, as instructions for that were not provided, but managed to fix it as well. The callback-url when registering the application on Imgur.com needs to be `imgurbar://oauth-callback`.

**Comment by ailinykh** [comment]
> I managed to fix this by editing line 48 in AppDelegate.swift to not replace the URL with "FAKE_CLIENT_ID", but I don't really know Swift so I'm not sure how to fix this from happening.

Yes, there is an option, but the preferable way is to pass it as an `Environment variable`.
To do that, select `Edit Scheme...`
<p align="center">
<img width="276" alt="Screenshot 2023-08-15 at 20 29 13" src="https://github.com/ailinykh/ImgurBar/assets/939390/8e311b9b-58d8-412a-98ca-9a2d560e537a">
</p>

choose `Run` in the sidebar menu and insert `client_id` as an `Environment variable`

<p align="center">
<img width="917" alt="Screenshot 2023-08-15 at 20 29 59" src="https://github.com/ailinykh/ImgurBar/assets/939390/7d549879-7228-4d76-beda-22117ace792e">
</p>

> I also ran into a callback-url error while logging in, as instructions for that were not provided

Yep, my fault. I will add it to the readme as well

> Even when logged into Imgur in ImageBar, all uploads are posted anonymously.

Correct. The feature is not completed yet. I hope #3 will fix this
